docs/framework/additional-apis/_httpresponse.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "_HttpResponse Field | Microsoft Docs"
+ms.custom: ""
+ms.date: "04/24/2017"
+ms.prod: ".net-framework"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology:
+  - "dotnet-winforms"
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+apiname:
+  - "System.Net"
```

Comment:
API Name needs to be the unique API identifier. So in this case it should be System.Net.HttpWebRequest._HttpResponse.